config.yaml

```diff
@@ -17,6 +17,10 @@ password:
 database:
 # Endpoint for scraping prometheus metrics. Default localhost:9090.
 metrics_endpoint: localhost:9090
+# Whether to use sql scripts supporting replication and sharding. Default false.
+replication:
+# Table names options should end with '_local'.
+# For global distributed tables name without '_local' is used.
 # Table with spans. Default "jaeger_spans_local".
```

Comment:
consider changing this to 

```
# Table with spans. Default "jaeger_spans_local" or `jaeger_spans` when replication is enabled.
```

and add a similar comment to all table names. The idea is to be explicit as much as possible.

Reply:
> No we cannot use script that uses `USE` because of [go-sql-driver/mysql#173 (comment)](https://github.com/go-sql-driver/mysql/issues/173#issuecomment-28539448).
> 
> We can make a requirement for user to create a database before starting jaeger, however we need to change the db name in scripts as I mentioned before.

@pavolloffay Then maybe we should keep "default" as the default database name.